dashboard/config/locales.yml

```diff
@@ -172,8 +172,7 @@
 "hy": "hy-AM"
 "hy-AM":
   english: "Armenian"
-  native: "Հայերեն"
-  debug: true
+  native: "հայերեն"
```

Comment:
Right. Not sure where the old one came from but checked more sources and the new one is more prevalent. Checking with our translator, but for now this is the most accurate as far as I can tell.